integrations/okta/nango.yaml

```diff
@@ -29,6 +29,12 @@ integrations:
                 input: OktaAssignRemoveUserGroup
                 scopes:
                     - okta.groups.manage
+            list-all-users:
+                description: Lists users in your org
+                output: OktaListUsersResponse
```

Comment:
can just be

`output: User`

models:
   User:
      id: string
      ...

Reply:
done